app/models/apple/publisher.rb

```diff
@@ -32,166 +34,213 @@ def episodes_to_sync
       show.episodes
     end
 
-    def poll!
+    def poll!(eps = episodes_to_sync)
```

Comment:
So what calls this?  (Or... what will eventually call this?)  A cron, or do we trigger from the publish-feed-job or something?

Reply:
Hrm, maybe I'm looking in the wrong place(s).  But I don't see anything calling `poll!` right now.

Reply:
Ahh, ok I thought you were talking about the parameters (which are symmetric with `#publish!`).

It's a workflow method. Called on the rails console to probe remote apple state. `publish!` is the superset method to this one and that is the method called from the job. But this one can be used in a manner that does not affect apple state.